src/nevermined/utils/WebServiceConnector.ts

```diff
@@ -123,6 +123,37 @@ export class WebServiceConnector extends Instantiable {
         return d
     }
 
+    public async getFile(
```

Comment:
The previous method `downloadFile` should be refactored to call `getFile` instead of having duplicated code